gradle/antlr.gradle

```diff
@@ -21,6 +21,7 @@
 apply plugin: 'antlr'
 
 generateGrammarSource {
+    outputDirectory = layout.buildDirectory.dir('generated-src/antlr/main/com/apple/foundationdb/relational/generated').get().asFile
```

Comment:
We'll have to think at some point how to make this more generic so that it accepts different package names, though for now, this should be fine. Something we will have to notice if we ever change the hardcoded package below

Reply:
Yeah, let's cross that bridge when we reach it then.